core-datastore/build.gradle.kts

```
@@ -19,6 +19,7 @@ import com.google.protobuf.gradle.generateProtoTasks
 import com.google.protobuf.gradle.protobuf
 import com.google.protobuf.gradle.protoc
 
+@kotlin.Suppress("DSL_SCOPE_VIOLATION")
```

Could we add a TODO with the upstream issue to track this? I believe this is the correct one: https://youtrack.jetbrains.com/issue/KTIJ-19369

Something like
```suggestion
// TODO: Remove once https://youtrack.jetbrains.com/issue/KTIJ-19369 is fixed
@kotlin.Suppress("DSL_SCOPE_VIOLATION")
```

---

It seems great. Thank you